### ISSUE TITLE
Audio in IDE

### ISSUE THREAD

**Issue by sanderaido** [lhartikk/ArnoldC#94]
This is probably the wrong place to post it, but it would be cool if some IDE would actually play the audio files each time you use something. After each autocomplete maybe?

**Comment by cesardeazevedo** [comment]
Yes. I did it, but the editor is on the web, I picked up all audio files from the movies and Arnold will narrate each line. 

check this out, @lhartikk what you think? :smile: 
http://cesardeazevedo.github.io/ArnoldC-Speaker/

**Comment by rranelli** [comment]
You sir, made my day.
